### ISSUE TITLE
Link "badssl.com" of the big hostname text back to the index

### ISSUE THREAD

**Issue by lgarron** [lgarron/badssl.com#60]
Styling should stay the same by default. Underline on hover/tap.

**Comment by hickford** [comment]
Yes please it would be useful to have a link back from the subdomains in the main site

**Comment by lgarron** [comment]
I've been experimenting with Jekyll templating for #59. That would be a good place to implement this.
